tests/e2e/test_e2e.py

```diff
@@ -361,3 +385,15 @@ def test_large_volume(client):
             abs_tol=FLOAT_TOLERANCE,
         ):
             break
+
+    # Test if values in wh store are correct
+    feature_retrieval_job = batch_client.get_batch_features(
+        entity_rows=customer_data[['datetime', 'customer_id']],
+        feature_ids=[
+            "customer_transactions_large:1:daily_transactions",
+            "customer_transactions_large:1:total_transactions",
```

Comment:
I really think we should be testing joins of multiple feature sets here. Can we please add that?

Reply:
See the test below this one

Reply:
That's a WIP, fixing some bugs wrt the joins before i push

Reply:
Ok, should we wait for that or do you want to merge this in?

Reply:
No, please dont merge this in, I'm still fixing the batch retrieval